src/utils/parseMarkdown.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+/**
+ * Parses simple markdown-like syntax and converts to React elements
+ *
+ * Supported features:
+ * - **text** for bold
+ * - Auto-detection of URLs (https://...) and converts them to clickable links
+ * - Preserves line breaks
+ *
+ * @param content - The text content to parse
+ * @param primaryColor - Color for links
+ * @returns React nodes with formatted content
+ */
+export const parseMarkdown = (content: string, primaryColor: string): React.ReactNode => {
```

Comment:
At some point we should consider using a real markdown parsing library but this felt like a lighter weight solution. It should be setup in a way that replacing should be easy in the future.